File: atv_pratica1/src/Moto.java
enum TipoPartida {
    ELETRICA,
    MANUAL
}

public class Moto extends Veiculo {
    private TipoPartida partida;
    private int cilidradas;


    public Moto() {
        super();
        this.cilidradas = 0;
        this.partida = TipoPartida.MANUAL;
    }


    public TipoPartida getPartida() {
        return partida;
    }


    @Override
    public String getTipo() {
        return "Moto";
    }

    public int getCilidradas() {
        return cilidradas;
    }

    public void setPartida(TipoPartida partida) {
        this.partida = partida;
    }

    public void setCilidradas(int cilidradas) {
        this.cilidradas = cilidradas;
    }
}

File: atv_pratica1_kotlin/src/main/kotlin/Moto.kt
enum class TipoPartida {
    ELETRICA,
    MANUAL
}

class Moto : Veiculo() {
    var partida: TipoPartida
    var cilidradas = 0

    init {
        partida = TipoPartida.MANUAL
    }

    override val tipo: String
        get() = "Moto"
}
